binding/tfe_utils.cc

```diff
@@ -237,7 +237,8 @@ void GetTFE_TensorHandleType(napi_env env, TFE_TensorHandle *handle,
   ENSURE_NAPI_OK(env, nstatus);
 }
 
-void AssignOpAttr(napi_env env, TFE_Op *tfe_op, napi_value attr_value) {
+void AssignOpAttr(napi_env env, TFE_Op *tfe_op, napi_value attr_value,
+                  tfnodejs::TF_ScopedStrings &scoped_strings) {
```

Comment:
Is a non-const reference argument appropriate? See https://google.github.io/styleguide/cppguide.html#Reference_Arguments

Reply:
I thought about it - but the `GetString()` method modifies the member `strings_refs_` which we can't do with a `const` signature.

Reply:
In which case should the argument be a pointer instead? (`tfnodejs::TF_ScopedStrings* scoped_strings`)

Reply:
Good idea - done.